pkg/services/sqlstore/user.go

```diff
@@ -454,7 +454,7 @@ func SearchUsers(query *models.SearchUsersQuery) error {
 	offset := query.Limit * (query.Page - 1)
 	sess.Limit(query.Limit, offset)
 	sess.Cols("u.id", "u.email", "u.name", "u.login", "u.is_admin", "u.is_disabled", "u.last_seen_at", "user_auth.auth_module")
-	sess.OrderBy("u.id")
+	sess.OrderBy("LOWER(u.email),u.email")
```

Comment:
@torkelo Should we have the same ordering here as for org_users (`sess.Asc("user.email", "user.login")`)?

Reply:
@aknuds1 Did you get a chance to review this change ?

Reply:
@vikkyomkar I'm waiting for consensus on ordering in #22371, then I'll need to know if we should apply the same ordering here.

Reply:
Hi @aknuds1 , now @torkelo has given his consent to this change pls refer  #22371